### PR TITLE
fixing

### PR DESCRIPTION
**Pull request by brotzeit**

### CODE REVIEW

File: rust_src/src/editfns.rs [hunk 1]
```diff
@@ -393,7 +393,7 @@ pub fn char_after(mut pos: LispObject) -> Option<EmacsInt> {
 /// Remaining arguments form a sequence of PROPERTY VALUE pairs for text
 /// properties to add to the result.
 /// usage: (fn STRING &rest PROPERTIES)
```
Comment (shaleh) [inline rust_src/src/editfns.rs]: Why are you dropping the min=1? That is accurate.

Comment (brotzeit) [reply rust_src/src/editfns.rs]: Fixed.

File: rust_src/src/marker.rs [hunk 2]
```diff
@@ -260,8 +254,8 @@ pub fn marker_insertion_type(marker: LispMarkerRef) -> bool {
 /// If ITYPE is non-nil, it means the marker advances when you insert text at it.
 /// If ITYPE is nil, it means the marker stays behind when you insert text at it.
 #[lisp_fn]
-pub fn set_marker_insertion_type(mut marker: LispMarkerRef, itype: LispObject) -> LispObject {
-    marker.set_insertion_type(itype.is_not_nil());
+pub fn set_marker_insertion_type(mut marker: LispMarkerRef, itype: bool) -> bool {
+    marker.set_insertion_type(itype);
     itype
```
Comment (brotzeit) [inline rust_src/src/marker.rs]: I just realized that itype will be returned as lisp bool and not as the original value. I think we shouldn't do this ?

Comment (shaleh) [reply rust_src/src/marker.rs]: Yeah, bad idea.

**Comment by brotzeit** [comment]
Should be ok.

**Pull request merged**